File: CS/WindowsFormsApplication5/PopupCalcEditFormDescendant.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using DevExpress.XtraEditors.Popup;
using DevExpress.XtraEditors;
using System.Drawing;

namespace WindowsFormsApplication5
{
    public class PopupCalcEditFormDescendant : PopupCalcEditForm
    {
        UserControl1 userControl;
        public PopupCalcEditFormDescendant(PopupBaseEdit ownerEdit)
            : base(ownerEdit)
        {
            userControl = new UserControl1();
           
        }
        public override void ShowPopupForm()
        {

            this.Height += userControl.Height;
            userControl.Dock = System.Windows.Forms.DockStyle.Bottom; 
            this.Controls.Add(userControl);
            base.ShowPopupForm();
        }
    }
}

File: VB/WindowsFormsApplication5/PopupCalcEditFormDescendant.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Text
Imports DevExpress.XtraEditors.Popup
Imports DevExpress.XtraEditors
Imports System.Drawing

Namespace WindowsFormsApplication5
	Public Class PopupCalcEditFormDescendant
		Inherits PopupCalcEditForm
		Private userControl As UserControl1
		Public Sub New(ByVal ownerEdit As PopupBaseEdit)
			MyBase.New(ownerEdit)
			userControl = New UserControl1()

		End Sub
		Public Overrides Sub ShowPopupForm()

			Me.Height += userControl.Height
			userControl.Dock = System.Windows.Forms.DockStyle.Bottom
			Me.Controls.Add(userControl)
			MyBase.ShowPopupForm()
		End Sub
	End Class
End Namespace
